### ISSUE TITLE
Auth context integrated with github signups and session info

### ISSUE THREAD

**Issue by evanmcneely** [oslabs-beta/GraphQL-Gateway#45]

**Comment by evanmcneely** [comment]
I have marked this as "on hold". There have been some issues with Github's incompatibility with an implicit OAth grant flow. Basically all OAuth requests coming from a browser are blocked by CORS. You can work around this by setting up a proxy server to handle Github OAuth requests but it seems a bit excessive to set up a proxy server just for Github. We could explore using passport.js. This is on hold because it is not a core feature of the product.

**Comment by evanmcneely** [comment]
The implementation is on branch `em/github` or `em/github-cors`
